.prettierignore

```diff
@@ -1,3 +1,5 @@
 .github
 dist
 test/unit/coverage
+.nuxt-storybook
```

Comment:
```suggestion
test/unit/coverage
.nuxt
.nuxt-storybook
```
The Nuxt build directory is also in the .gitignore, it's a good time to include it as it's also autogenerated.